ui/sv_icons.py

```diff
@@ -0,0 +1,59 @@
+import bpy
+import os
+import glob
+import bpy.utils.previews
+
+DEBUG = False
+def logDebug(message, extra=""):
+    if DEBUG:
+        print(message, extra)
+
+# global variable to store icons in
+_custom_icons = None
+
+def customIcon(name):
+    logDebug("customIcon called: ", name)
+    global _custom_icons
```

Comment:
dicts are global to the file, hence this line won't be needed
```diff
-    global _custom_icons
```
if doesn't matter where in Sverchok the customIcon function is called , code inside the function will first look at the file it exists in, and it will find the `_custom_icons` dict. and it will be read/writable without using the `global` keyword.

You can leave it as it is, but i _will_ change it.